src/material-examples/tooltip-manual/tooltip-manual-example.html

```diff
@@ -1,19 +1,19 @@
 <div>
-  <span> Mouse over to </span>
+  <span> Click the following buttons to... </span>
   <button mat-button
-          (mouseenter)="tooltip.show()"
+          (click)="tooltip.show()"
```

Comment:
Based on the new behavior, I think the `aria-label` attributes should be updated to just say, e.g.
"Show tooltip on the button at the end of this section"